deploy-service/teletraanservice/src/main/java/com/pinterest/teletraan/ConfigHelper.java

```diff
@@ -140,8 +140,8 @@ public static TeletraanServiceContext setupContext(TeletraanServiceConfiguration
         HashMap<String, SourceControlManager> managers = new HashMap<String, SourceControlManager>();
         for(SourceControlFactory scf : sourceControlConfigs) {
             SourceControlManager scm = scf.create();
-            String type = scm.getType();
-            managers.put(type, scm);
+            String type = scm.getTypeName();
+            managers.put(type, scm);  
```

Comment:
remove trailing whitespace (might be an option for this in your IDE)

Reply:
ack